bundles/statistics/statsgrid2016/resources/css/seriesplayback.css

```diff
@@ -0,0 +1,184 @@
+
+/* ********************************************
+ * Statsgrid series control
+ * ******************************************* */
+.statsgrid-series-control-plugin {
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(0, 0, 0, 0.2);
+  white-space: nowrap;
+  z-index: 15002!important;
+}
+.statsgrid-series-control-plugin .flyout-resizer {
+  cursor: ew-resize;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+.statsgrid-series-control-container {
+  width: 100%;
+  min-width: 290px;
+  font-size: 12px; 
+  overflow: hidden;
+}
+.statsgrid-series-control-plugin,
+.statsgrid-series-control-container {
+  min-width: 290px;
+}
+.statsgrid-series-control-container .value-controls {
+  text-align: left;
+  font-size: 12px;
+}
+.stats-series-speed {
+  display: inline-block;
+  vertical-align: baseline;
+}
+.stats-series-speed label {
+  display: block;
+}
+.stats-series-speed select {
+    width: 100px;
+    height: 24px;
+    padding-top: 0;
+    padding-bottom: 0;
+}
+
+.line-svg g.drag-handle circle{
```

Comment:
These are a bit too generic. Add .statsgrid-series-control-plugin in front of them to narrow the scope.